clm_data/README.md

```diff
@@ -1,21 +1,107 @@
+
+- **Key Variables**
+    - `medlynslope`: Medlyn slope of conductance-photosynthesis relationship
+    - `medlynintercept`: Medlyn intercept of conductance-photosynthesis relationship
+
+- **Data Download** The dataset can be downloaded
+[here](https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/paramdata/clm5_params.c171117.nc)
+
+
+## Output Files
+
+### 1. `dominant_PFT_map.nc`
+Contains the dominant PFT for each grid cell on the global grid.
+- **Purpose**: Used as an input file for mapping the physiological parameters to the global grid based on the dominant PFT.
+
+### 2. `vegetation_properties_map.nc`
+Contains the mapped vegetation properties for each grid cell.
+- **Contents**:
+  - `medlynslope(lat, lon)`: Medlyn slope of conductance-photosynthesis relationship (kPa^0.5)
+  - `medlynintercept(lat, lon)`: Medlyn intercept of conductance-photosynthesis relationship (umol m^-2 s^-1)
+  - `taulnir(lat, lon)`: Leaf transmittance: near-IR (fraction)
+  - `taulvis(lat, lon)`: Leaf transmittance: visible (fraction)
+  - `tausnir(lat, lon)`: Stem transmittance: near-IR (fraction)
+  - `tausvis(lat, lon)`: Stem transmittance: visible (fraction)
+  - `vcmx25(lat, lon)`: Maximum rate of carboxylation (umol CO2/m**2/s)
+
+### 3. `mechanism_map.nc`
+Contains two variables which describe the photosynthesis mechanism for each grid cell.
+`c3_dominant` has a value of 1.0 when C3 is the dominant photosynthesis mechanism for that grid cell and a
+value of 0.0 if C4 is dominant. `c3_proportion` has values in [0.0, 1.0] which represent the proportion of plants
+in a grid cell that use C3 photosynthesis.
+- **Contents**:
+  - `c3_dominant(lat, lon)` (0.0 and 1.0)
+  - `c3_proportion(lat, lon)` (0 to 1)
+
+## Scripts
+
+### 1. `dominant_pft.py`
+- **Description**: Script to determine the dominant PFT for each grid cell.
+- **Purpose**: Processes the surface data to create the `dominant_PFT_map.nc` file.
+
+### 2. `pft_variables.py`
+- **Description**: Main script for mapping vegetation properties based on the dominant PFT.
+- **Functionality**:
+  - Reads the dominant PFT for each grid cell from `dominant_PFT_map.nc`.
+  - Reads physiological parameters from `pft-physiology.c110225.nc`.
+  - Maps these parameters to the global grid based on the dominant PFT.
+  - Outputs the mapped parameters to `vegetation_properties_map.nc`.
+
+### 3. `dominant_mechanism.py`
+- **Description**: Script to determine the dominant photsynthesis mechanism for each grid cell.
+- **Functionality**:
+  - Reads the percent plant functional type for each grid cell (% of total plants in cell) from `surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc`.
+  - If the PFT with the highest percentage in a cell is not PFT #14, then the cell is marked as C3 dominant, and a 1.0 is placed in the grid cell in the output parameter `c3_dominant`. If the highest percentage is PFT #14, a value of 0.0 is placed in the grid cell.
```

Comment:
Can you please add more explanation about this? It sounds arbitrary. What's the motivation behind picking number 14?

Reply:
PFT 14 refers to C4 grasses (the only PFT that follows under C4 photosynthesis). The full list of PFTs are: 
![Screenshot 2024-09-18 at 10 36 28 AM](https://github.com/user-attachments/assets/cb4a75d7-5b11-423c-939c-0f9c118c7a96)

Please refer to: chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/https://www2.cesm.ucar.edu/models/cesm2/land/CLM50_Tech_Note.pdf